source/src/protocols/cyclic_peptide_predict/SimpleCycpepPredictApplication.cc

```diff
@@ -2160,6 +2213,39 @@ SimpleCycpepPredictApplication::find_first_and_last_thioether_lariat_residues(
 	}
 }
 
+/// @brief Given a pose, find the first and last lanthipeptide bond-forming residues.
+/// @details Randomly choose based on input residue numbers
+void
+SimpleCycpepPredictApplication::find_first_and_last_lanthipeptide_residues(
```

Comment:
Slight nomenclature note: "find" doesn't really mesh with random choice in my mind (finding tends to lean more toward the deterministic search). I would personally go with "select" or something like that.

Reply:
This keeps the code consistent with other functions in the application.